src/VisualStudio/Core/Test/ProjectSystemShim/VisualBasicCompilerOptionsTests.vb

```diff
@@ -259,6 +259,15 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim
 
                 outputs = CType(environment.Workspace.GetCompilationOutputs(project.Test_VisualStudioProject.Id), CompilationOutputFilesWithImplicitPdbPath)
                 Assert.Equal("C:\NewFolder3\test3.dll", outputs.AssemblyFilePath)
+
+                ' Relative path - set by VBIntelliProj in VB Web App project
```

Comment:
Worth doing as a separate test method since this one is a bit special?

Reply:
I need it to test that the previously set value is cleared (`Nothing`), so it's convenient to have it after the previous cases.